### PR TITLE
Fixes for cli segment

### PR DESCRIPTION
**Pull request by hasan7n**
Fixes issue #(Enter issue number here)

## Proposed Changes
  -
  -
  -

### CODE REVIEW

**Comment by hasan7n** [comment]
Result of running: `FeTS_CLI_Segment -d /data_renamed -a fets_singlet,fets_triplet -lF STAPLE,ITKVoting,SIMPLE,MajorityVoting -g 0 -t 0`:

```
root@3bb2af1add88:/Front-End# tree /data_renamed/
/data_renamed/
|-- BraTS-GLI-00001-001
|   |-- BraTS-GLI-00001-001_flair.nii.gz
|   |-- BraTS-GLI-00001-001_t1.nii.gz
|   |-- BraTS-GLI-00001-001_t1ce.nii.gz
|   |-- BraTS-GLI-00001-001_t2.nii.gz
|   `-- SegmentationsForQC
|       |-- BraTS-GLI-00001-001_fused_itkvoting_seg.nii.gz
|       |-- BraTS-GLI-00001-001_fused_majorityvoting_seg.nii.gz
|       |-- BraTS-GLI-00001-001_fused_simple_seg.nii.gz
|       |-- BraTS-GLI-00001-001_fused_staple_seg.nii.gz
|       |-- fets_singlet
|       |   |-- BraTS-GLI-00001-001
|       |   |   |-- BraTS-GLI-00001-001_fets_singlet_seg.nii.gz
|       |   |   `-- _t1.nii.gz
|       |   `-- _fets_singlet_subdirs_to_scores.pkl
|       |-- fets_singlet.nii.gz
|       |-- fets_triplet
|       |   |-- BraTS-GLI-00001-001
|       |   |   |-- BraTS-GLI-00001-001_fets_triplet_seg.nii.gz
|       |   |   `-- _t1.nii.gz
|       |   `-- _fets_triplet_subdirs_to_scores.pkl
|       `-- fets_triplet.nii.gz
|-- BraTS-GLI-00015-000
|   |-- BraTS-GLI-00015-000_flair.nii.gz
|   |-- BraTS-GLI-00015-000_t1.nii.gz
|   |-- BraTS-GLI-00015-000_t1ce.nii.gz
|   |-- BraTS-GLI-00015-000_t2.nii.gz
|   `-- SegmentationsForQC
|       |-- BraTS-GLI-00015-000_fused_itkvoting_seg.nii.gz
|       |-- BraTS-GLI-00015-000_fused_majorityvoting_seg.nii.gz
|       |-- BraTS-GLI-00015-000_fused_simple_seg.nii.gz
|       |-- BraTS-GLI-00015-000_fused_staple_seg.nii.gz
|       |-- fets_singlet
|       |   |-- BraTS-GLI-00015-000
|       |   |   |-- BraTS-GLI-00015-000_fets_singlet_seg.nii.gz
|       |   |   `-- _t1.nii.gz
|       |   `-- _fets_singlet_subdirs_to_scores.pkl
|       |-- fets_singlet.nii.gz
|       |-- fets_triplet
|       |   |-- BraTS-GLI-00015-000
|       |   |   |-- BraTS-GLI-00015-000_fets_triplet_seg.nii.gz
|       |   |   `-- _t1.nii.gz
|       |   `-- _fets_triplet_subdirs_to_scores.pkl
|       `-- fets_triplet.nii.gz
`-- logs

13 directories, 32 files
```

File: src/applications/FeTS_CLI_Segment.cxx [hunk 1]
```diff
@@ -135,7 +135,7 @@ int main(int argc, char** argv)
     hardcodedModelWeightPath = hardcodedOpenFLPath + "/bin/federations/weights/", // start with the common location
     hardcodedPythonPath = hardcodedOpenFLPath + "/venv/bin/python", // this needs to change for Windows (wonder what happens for macOS?)
     hardcodedPythonPath_fusion = fetsApplicationPath + "/LabelFusion/venv/bin/python", // this needs to change for Windows (wonder what happens for macOS?)
-    scriptToCall = hardcodedOpenFLPath + "/submodules/fets_ai/Algorithms/fets/bin/brainmage_validation_scores_to_disk.py"; // the script that does the inference and scoring
+    scriptToCall = hardcodedOpenFLPath + "/submodules/fets_ai/Algorithms/fets/bin/brainmage_validation_outputs_to_disk_newer.py"; // the script that does the inference and scoring
```
Comment (sarthakpati) [inline src/applications/FeTS_CLI_Segment.cxx:138]: I don't think we have this new script in the PR, right?

Comment (hasan7n) [reply src/applications/FeTS_CLI_Segment.cxx:138]: The script is already in the submodules.
This script is here: https://github.com/FeTS-AI/Algorithms/blob/10383517577386dac20e8bf6bafc6dd43898ce05/fets/bin/brainmage_validation_outputs_to_disk_newer.py

it's in the Algorithms submodule of the OpenFederatedLearning submodule. Is that what you mean?

Comment (hasan7n) [reply src/applications/FeTS_CLI_Segment.cxx:138]: Note that the previous script `brainmage_validation_scores_to_disk.py` does inference and scoring, but it does NOT write the prediction file to the disk; it only writes the scores. `FeTS_CLI_Segment` expects prediction files to do fusion.

On the other hand, `brainmage_validation_outputs_to_disk_newer.py` seemed like a script that has the same interface as the previous one, its logic is very similar, and it writes the predictions to disk. Seemed to me like it was the script intended to be used.

Comment (sarthakpati) [reply src/applications/FeTS_CLI_Segment.cxx:138]: Thanks! You are right on both accounts.

**Review by sarthakpati — APPROVED** [review]
LGTM

**Pull request merged**